app/models/user.rb

```diff
@@ -23,4 +23,14 @@ class User < ApplicationRecord
   has_many(:ratings, { :class_name => "Rating", :foreign_key => "rater_id", :dependent => :nullify })
   has_many(:restaurants, { :class_name => "Restaurant", :foreign_key => "owner_id", :dependent => :destroy })
   has_many(:rated_burritos, { :through => :ratings, :source => :burrito })
+
+ def recommendation
```

Comment:
If this method is supposed to return a list of multiple burritos, I might make the method name plural instead of singular to make it more clear.